### PR TITLE
Refactor examples

### PR DESCRIPTION
**Pull request by KaiVolland**
This refactors the examples and moves them to the component specific folder.

### CODE REVIEW

File: tasks/build-examples.js [hunk 1]
```diff
@@ -5,33 +5,67 @@ var markdown = require('metalsmith-markdown');
...
 
-var srcDir = path.join(__dirname, '..', 'examples');
+var srcDir = path.join(__dirname, '..', 'src');
 var destDir = path.join(__dirname, '..', 'build', 'examples');
 var tplDir = path.join(__dirname, '..', 'example-templates');
 
-function augmentExamples(files, metalsmith, done) {
-  setImmediate(done); // all remaining code is synchronous
+/**
+ * Fixes the import string in the example source to use 'react-gepo' instead of
```
Comment (ahennr) [inline tasks/build-examples.js]: typo in `react-geo`

File: webpack.examples.config.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 const commonConfig = require('./webpack.common.config.js');
 
 commonConfig.entry = {
-  userChip: './examples/userChip.js'
+  UserChip: './src/UserChip/UserChip.example.jsx'
 };
 
 commonConfig.output = {
-  filename: '[name].js',
+  filename: '[name]/[name].js',
```
Comment (ahennr) [inline webpack.examples.config.js]: Does this also work for components that are nested more deeply, e.g. `ModuleType/ModuleA/ModuleA.jsx` or don't we have such a nested structure in the examples?

Comment (KaiVolland) [reply webpack.examples.config.js]: Should work for the most cases. We might want to improve this later.

**Comment by KaiVolland** [comment]
thx

**Pull request merged**